content/config/filter.md

```diff
@@ -0,0 +1,114 @@
+```
+
+The key represents the field that you want to filter on.
+The value is the pattern that the specified field must match.
+The operator can be either equals (`=`) or not equals (`!=`).
+
+For example, the command `docker images --filter reference=alpine` filters the
+output of the `docker images` command to only print `alpine` images.
+
+```console
+$ docker images
+REPOSITORY   TAG       IMAGE ID       CREATED          SIZE
+ubuntu       20.04     33a5cc25d22c   36 minutes ago   101MB
+ubuntu       18.04     152dc042452c   36 minutes ago   88.1MB
+alpine       3.16      a8cbb8c69ee7   40 minutes ago   8.67MB
+alpine       latest    7144f7bab3d4   40 minutes ago   11.7MB
+busybox      uclibc    3e516f71d880   48 minutes ago   2.4MB
+busybox      glibc     7338d0c72c65   48 minutes ago   6.09MB
+$ docker images --filter reference=alpine
+REPOSITORY   TAG       IMAGE ID       CREATED          SIZE
+alpine       3.16      a8cbb8c69ee7   40 minutes ago   8.67MB
+alpine       latest    7144f7bab3d4   40 minutes ago   11.7MB
+```
+
+The available fields (`reference` in this case) depends on the command you run.
+Some filters expect an exact match. Others handle partial matches. Some filters
+let you use regular expressions.
+
+Refer to the [CLI reference description](#reference) for each command to learn
+about the supported filtering capabilities for each command.
+
+## Combining filters
+
+You can combine multiple filters by passing multiple `--filter` flags. The
+following example shows how to print all images that don't match
+`alpine:latest` or `busybox`.
+
+```console
+$ docker images
+REPOSITORY   TAG       IMAGE ID       CREATED       SIZE
+ubuntu       20.04     33a5cc25d22c   2 hours ago   101MB
+ubuntu       18.04     152dc042452c   2 hours ago   88.1MB
+alpine       3.16      a8cbb8c69ee7   2 hours ago   8.67MB
+alpine       latest    7144f7bab3d4   2 hours ago   11.7MB
+busybox      uclibc    3e516f71d880   2 hours ago   2.4MB
+busybox      glibc     7338d0c72c65   2 hours ago   6.09MB
+$ docker images --filter reference=alpine:latest --filter=reference=busybox
+REPOSITORY   TAG       IMAGE ID       CREATED       SIZE
+alpine       latest    7144f7bab3d4   2 hours ago   11.7MB
+busybox      uclibc    3e516f71d880   2 hours ago   2.4MB
+busybox      glibc     7338d0c72c65   2 hours ago   6.09MB
+```
+
+### Multiple negated filters
+
+Some commands support negated filters on [labels](./labels-custom-metadata.md).
+Negated filters only consider results don't match the specified patterns. The
+following command prunes all containers that aren't labeled `foo`.
+
+```console
+$ docker container prune --filter "label!=foo"
+```
+
+There's a catch in combining multiple negated label filters. Using multiple
+negated filters creates a single negative constraint. The following command
+prunes all containers except those labeled both `foo` and `bar`. Containers
+labeled either `foo` or `bar`, but not both, will be pruned.
```

Comment:
I think making it clear that this forms a logical `AND` would be a bit easier to parse?